book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'rental'
```

Comment:
**[Optional]**

- [ ] It would be nice if you rename **rental** to **rentals** because you have a file with an `s` at the end so it is breaking the operation :thinking: and you are calling it with the wrong name means without `s` at the end :fearful: Kindly put an `s` at the end of `require_relative 'rental'`

Reply:
Hi @HAMMAS-SALEEM ,  I'm immensely grateful for the detailed feedback! All changes have been duly implemented.  Please, I would appreciate it if you could help verify all is done before I finally submit it for review, as this is my last chance. Many thanks!